### PR TITLE
Sync on ctrl+s

### PR DESCRIPTION
**Pull request by Compositr**
Addresses [Bug]: ctrl+s should sync #1685

Adds a `KeyHandlers` to `SyncButton` to capture ctrl+s

### CODE REVIEW

File: packages/desktop-client/src/components/Titlebar.tsx [hunk 1]
```diff
@@ -173,30 +174,40 @@ export function SyncButton({ style, isMobile = false }: SyncButtonProps) {
...
-          ? 'Offline'
-          : 'Sync'}
-      </Text>
-    </Button>
+    <>
+      <KeyHandlers
+        keys={{
+          'ctrl+s': () => {
+            sync();
+          },
```
Comment (MatissJanis) [inline packages/desktop-client/src/components/Titlebar.tsx:182]: **🥜 nitpick:** ‏
```suggestion
          'ctrl+s': sync,
```

**Review by MatissJanis — COMMENTED** [review]
I think it would also make sense to add `cmd+s` support for this for mac users. WDYT?

**Comment by shaankhosla** [comment]
> I think it would also make sense to add `cmd+s` support for this for mac users. WDYT?

I think that's necessary, on my mac I don't see the Sync button and ctrl+s doesn't work.
image

**Comment by Compositr** [comment]
> > I think it would also make sense to add `cmd+s` support for this for mac users. WDYT?
> 
> I think that's necessary, on my mac I don't see the Sync button and cmd+s doesn't work. image

I think the sync button only appears when you're using a server (the demo file has no server) which is expected behaviour afaik.

I'll look into Mac command key support though although I don't have a Mac so I'm not quite sure how to emulate the command key.

File: packages/desktop-client/src/components/Titlebar.tsx [hunk 2]
```diff
@@ -177,9 +177,7 @@ export function SyncButton({ style, isMobile = false }: SyncButtonProps) {
     <>
       <KeyHandlers
         keys={{
-          'ctrl+s': () => {
-            sync();
-          },
+          'ctrl+s': sync,
```
Comment (Compositr) [inline packages/desktop-client/src/components/Titlebar.tsx]: Turns out KeyHandler's props doesn't like this, so I've reverted this in 81446614a0c3f5c945d96d7d3cfbe37e25d64af7

**Review by MatissJanis — APPROVED** [review]

**Pull request merged**